src/style.css

```diff
@@ -1 +1,95 @@
 /* add styles here */
+* {
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.form--big,
+.form--small {
+  position: relative;
+  margin-top: 20px;
+  transition: all 0.3s ease;
+}
+
+.input {
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
it's better to set in `*` selector

Reply:
it shouldn't break anything. At least, if everything looks the same. Anyway, let's keep it as it is, just take it into account in the future